lms/templates/video.html

```diff
@@ -33,12 +32,10 @@ <h4 class="video-error hidden">${_('No playable video sources found.')}</h4>
                   <div class="secondary-controls"></div>
               </div>
           </section>
-          <a class="nav-skip sr" id="before-transcript_${id}" href="#after-transcript_${id}">${_('Skip to end of transcript.')}</a>
```

Comment:
@clrux  Does this mean that there is no longer the capability to skip to the end of a transcript?   What will the impact of that be on users?